.travis.yml

```diff
@@ -17,11 +17,12 @@ matrix:
     - env: PGVERSION=10
     - env: PGVERSION=11
     - env: PGVERSION=12
+    - env: LINTING=true
 before_install:
-  - git clone -b v0.7.12 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b update-uncrustify-on-travis --depth 1 https://github.com/citusdata/tools.git
   - sudo make -C tools install
-  - setup_apt
-  - nuke_pg
+  - 'if [ "$LINTING" == "" ]; then setup_apt; fi'
+  - 'if [ "$LINTING" == "" ]; then nuke_pg; fi'
```

Comment:
Another way to write that is to use the built-in test `-z` and `-n` operations, which you might prefer?

In a shell here where $LINTING is not defined here's what I have:

``` shell
$ help test
...
        -z STRING      True if string is empty.    
        -n STRING
        STRING         True if string is not empty.
...

$ test -n "$LINTING" || echo ok
ok

$ [ -n "$LINTING" ] || echo ok
ok
```